tests/acceptance/config/behat.yml

```diff
@@ -414,6 +414,15 @@ default:
         - CollaborationContext:
         - TrashbinContext:
 
+    apiVersions:
+      paths:
+        - "%paths.base%/../features/apiVersions"
+      context: *common_ldap_suite_context
+      contexts:
```

Comment:
why separate suite. we already have some file version tests. maybe create new feature file in one of the exisitng suite?

Reply:
@saw-jan could you give suite name because i couldn't find the dedicated suite for versioning

Reply:
https://github.com/owncloud/ocis/tree/master/tests/acceptance/features/coreApiVersions

Reply:
Even though same feature. The way ocis disable file versioning is different than core. Isn't it ?
IMO we should keep it separate rather than mixing with core suite.

Reply:
the word `coreApi` deceives us, tests in ocis both `apiX` and `coreApiX` are for the ocis server and not for the oc10. So, I don't see a reason not to add feature file there.

Reply:
And we should rename all `coreApiX` to `apiX` soon

Reply:
> And we should rename all `coreApiX` to `apiX` soon

I mean same thing

Reply:
But I prefer not to add new suite in this PR. Adding new feature file in `coreApiVersions` should be fine

Reply:
i will put my PR on hold until rename complete

Reply:
we don't have planned that yet. It should be done with proper planning. So, please move this PR forward